bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/EventCondition.cpp

```diff
@@ -0,0 +1,70 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Core/Types/Shared.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/EventCondition.hpp>
+
+using namespace pybind11;
+
+using ostk::core::types::Shared;
+using ostk::core::types::String;
+using ostk::core::types::Real;
+
+using ostk::math::obj::VectorXd;
+
+using ostk::astro::EventCondition;
+
+// Trampoline class for virtual member functions
+class PyEventCondition : public EventCondition
+{
+   public:
+    using EventCondition::EventCondition;
+
+    // Trampoline (need one for each virtual function)
+
+    bool isSatisfied(const Real& currentValue, const Real& previousValue) const override
+    {
+        PYBIND11_OVERRIDE(bool, EventCondition, isSatisfied, currentValue, previousValue);
+    }
+
+    Real evaluate(const VectorXd& aStateVector, const Real& aTime) const override
+    {
+        PYBIND11_OVERRIDE_PURE(Real, EventCondition, evaluate, aStateVector, aTime);
+    }
```

Comment:
I guess there will be a usage when adding more later. Do we need it in here? (and the associated binding function)?

Reply:
This is required if we want to be able to define new EventConditions in Python

Reply:
Aaaaah, I see that makes sense! thanks for the clarification